### PR TITLE
LPS-155766

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://issues.liferay.com/browse/LPS-155766

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/test/java/com/liferay/dynamic/data/mapping/form/builder/internal/converter/DDMFormRuleToDDMFormRuleModelConverterTest.java [hunk 1]
```diff
@@ -379,32 +381,58 @@ private List<String> _extractCallFunctionParameters(String callFunction) {
...
+		DDMExpressionFunctionTracker ddmExpressionFunctionTracker =
+			Mockito.mock(DDMExpressionFunctionTracker.class);
 
-		field.set(
-			_ddmExpressionFactoryImpl, new DDMExpressionFunctionTrackerImpl());
+		Mockito.when(
+			ddmExpressionFunctionTracker.getDDMExpressionFunctionFactories(
+				Matchers.any())
+		).thenReturn(
+			HashMapBuilder.<String, DDMExpressionFunctionFactory>put(
```
Comment (rodrigopaulino) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/test/java/com/liferay/dynamic/data/mapping/form/builder/internal/converter/DDMFormRuleToDDMFormRuleModelConverterTest.java:391]: Do you think we can buy lines here? Perhaps create a method that iterates a collection of keys and puts the same value into the map? If you think this isn't worth it, I trust your judgement and we can move forward with this. 😄

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/ValidateExpressionBuilderContentMVCResourceCommand.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+ */
+@Component(
+	immediate = true,
+	property = {
+		"javax.portlet.name=" + ObjectPortletKeys.OBJECT_DEFINITIONS,
+		"mvc.command.name=/object_definitions/validate_expression_builder_content"
+	},
+	service = MVCResourceCommand.class
+)
+public class ValidateExpressionBuilderContentMVCResourceCommand
```
Comment (rodrigopaulino) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/ValidateExpressionBuilderContentMVCResourceCommand.java:46]: Can we name this command "ValidateExpression"?

**Review by rodrigopaulino — COMMENTED** [review]
Hey, @carolmariaabb. The tests commit is so satisfying!! ☺️ 
Great job here. I just have a couple of suggestions. Let me know what you think. 👏🏻

**Comment by carolmariaabb** [comment]
Hi @rodrigopaulino,
I made the suggested changes. Please let me know what you think 😄

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by rodrigopaulino — APPROVED** [review]
LGTM! 👏🏻

**Comment by rodrigopaulino** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**